### PR TITLE
Issue #14: Updated Readme File

### PR DESCRIPTION
**Pull request by dhawal-793**
Added Vite Pnpm and Vercel in the Project use section.

Updated Preview Image.

preview-image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div id="header" align="center">
     <h1>🔥 Free-Hit </h1>
     <h3><strong><em>Your search ends here 🔍</em></strong></h3>
-  <img src="https://i.ibb.co/kB8J5gy/Screenshot-2023-03-10-4-21-07-PM.png" alt="home page of the website"<br>
+  <img src="https://user-images.githubusercontent.com/88284372/229340283-b9f74f44-d41c-4690-a6c1-f93333b61e9e.png" alt="home page of the website"<br>
```
Comment (priyankarpal) [inline README.md:4]: add `>` 

``` suggestion
  <img src="https://user-images.githubusercontent.com/88284372/229340283-b9f74f44-d41c-4690-a6c1-f93333b61e9e.png" alt="home page of the website"><br>
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by dhawal-793** [comment]
@priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.

**Comment by priyankarpal** [comment]
> @priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.

which blog website?

**Comment by dhawal-793** [comment]
> > @priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.
> 
> which blog website?

You have mentioned blogs on your github

**Comment by priyankarpal** [comment]
> > > @priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.
> > 
> > 
> > which blog website?
> 
> You have mentioned blogs on your github

oh! I write on hashnode(currently inactive) & I used gh action to show the blogs on my GitHub README file.

**Comment by dhawal-793** [comment]
> > > > @priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.
> > > 
> > > 
> > > which blog website?
> > 
> > 
> > You have mentioned blogs on your github
> 
> oh! I write on hashnode(currently inactive) & I used gh action to show the blogs on my GitHub README file. https://itspp.hashnode.dev/

Yes, so where do you store the blogs, I am working on next Js blog website, but I am having some doubts that's why I asked.

**Comment by priyankarpal** [comment]
> > > > > @priyankarpal can you share your blog website repository url, I want to see the tech stack, and the file structure.
> > > > 
> > > > 
> > > > which blog website?
> > > 
> > > 
> > > You have mentioned blogs on your github
> > 
> > 
> > oh! I write on hashnode(currently inactive) & I used gh action to show the blogs on my GitHub README file. https://itspp.hashnode.dev/
> 
> Yes, so where do you store the blogs, I am working on next Js blog website, but I am having some doubts that's why I asked.

only hashnode. I don't use any hosting platform to store my blogs.

**Comment by dhawal-793** [comment]
Okey.

**Comment by JasonDsouza212** [comment]
Everything looks good, but please don't change the screenshot as the website looks the same as it did earlier. If you can make that change, I can merge it.

**Comment by dhawal-793** [comment]
> Everything looks good, but please don't change the screenshot as the website looks the same as it did earlier. If you can make that change, I can merge it.

The current preview image doesn't show all the available options, that's why I changed it, you can check once again.

After that, If you say I will remove it.

**Comment by JasonDsouza212** [comment]
That's a good thing that you did, but the UI in my system doesn't look like the screenshot you uploaded
Screenshot from 2023-04-02 18-26-27

We can get this UI at 100% zoom

**Comment by dhawal-793** [comment]
> That's a good thing that you did, but the UI in my system doesn't look like the screenshot you uploaded Screenshot from 2023-04-02 18-26-27
> 
> We can get this UI at 100% zoom

I have updated with the image you provided.

**Comment by JasonDsouza212** [comment]
Perfect, @dhawal-793  Keep up the good work, and thank you for your contributions

**Review by JasonDsouza212 — APPROVED** [review]

**Pull request merged**